packages/react-noop-renderer/src/createReactNoop.js

```diff
@@ -854,6 +870,16 @@ function createReactNoop(reconciler: Function, useMutation: boolean) {
       return yieldedValues;
     },
 
+    flushPassiveEffects() {
+      // Trick to flush passive effects without exposing an internal API:
+      // Create a throwaway root and schedule a dummy update on it.
+      const rootID = 'bloop';
```

Comment:
tiny nit: This is unlikely to conflict, but should we maybe pick something a little longer and more unique?

Reply:
I gotcha